### PR TITLE
Mobile pricing page

### PR DESCRIPTION
**Pull request by sleaper**
- [x] - fix the pricing page link
- [x] - cleaning useless files

### CODE REVIEW

**Comment by sleaper** [comment]
@capaj Do you know if I can use react-native-code-push to update .env variables? Because we use react-native-config, which exports .env file to JS.

**Comment by capaj** [comment]
yes you absolutely can

**Review by capaj — APPROVED** [review]
LGTM. Ideally next time open the cleanup of unused files as separate PR, but not a blocker

File: mobile-app/src/screens/Account/Limits.tsx [hunk 1]
```diff
@@ -11,12 +11,38 @@ import {
...
+  const handlePress = useCallback(async () => {
+    // Checking if the link is supported for links with custom URL scheme.
+    const supported = await Linking.canOpenURL(url)
+
+    if (supported) {
+      // Opening the link with some app, if the URL scheme is "http" the web link should be opened
+      // by some browser in the mobile
+      await Linking.openURL(url)
+    } else {
+      console.error(`Don't know how to open this URL: ${url}`)
```
Comment (capaj) [inline mobile-app/src/screens/Account/Limits.tsx:35]: showing this as notification to the user would be better

**Pull request merged**